src/backend/CassandraBackend.cpp

```diff
@@ -1496,6 +1580,16 @@ CassandraBackend::open(bool readOnly)
         if (!selectNFT_.prepareStatement(query, session_.get()))
             continue;
 
+        query.str("");
+        query << "SELECT token_id,sequence,owner,is_burned"
+              << " FROM " << tablePrefix << "nf_tokens WHERE"
+              << " token_id IN ? AND"
+              << " sequence <= ?"
+              << " ORDER BY sequence DESC"
+              << " PER PARTITION LIMIT 1";
```

Comment:
this query could actually replace the existing nf_token select query used for `nft_info`. not sure we really need to save on memory but it's possible.

Reply:
Not as part of this PR - feel free to add a comment in the relevant place in nft_info handlers tho.